skill-tree/src/test/java/com/RDS/skilltree/unit/EndorsementServiceTest.java

```diff
@@ -602,4 +618,126 @@ void testCreateEndorsementWithInvalidSkill() {
         // Verify that save method is not called
         verify(endorsementRepository, never()).save(any(EndorsementModel.class));
     }
+
+    @Test
+    @Disabled
+    @DisplayName("Return unauthorized given user is not a super user")
```

Comment:
nit: Test case name can be clearer & specific to patch endorsement status.

Reply:
changed [here](https://github.com/Real-Dev-Squad/skill-tree-backend/pull/117/commits/3fec27986459c7bbb9c8b532651be564e7eb52d4)

Do let me know if it needs to be changed